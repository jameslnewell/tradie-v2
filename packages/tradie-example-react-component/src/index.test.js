import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import Component from './';

describe('Component', () => {
  describe('render()', () => {
    it('should render', () => {
      const renderer = ReactTestRenderer.create(<Component />);
      expect(renderer.toJSON()).toMatchSnapshot();
    });
  });
});
