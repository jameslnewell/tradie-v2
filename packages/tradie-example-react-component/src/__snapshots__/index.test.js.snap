// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component render() should render 1`] = `
<button
  className="sc-bdVaJa dHVegr"
>
  Hello Worlds!
</button>
`;
